sentry_sdk/integrations/arq.py

```diff
@@ -116,7 +118,14 @@ async def _sentry_run_job(self, job_id, score):
                 origin=ArqIntegration.origin,
             ) as span:
                 return_value = await old_run_job(self, job_id, score)
-                span.set_status(SPANSTATUS.OK)
+
+                status_unset = (
+                    hasattr(span._otel_span, "status")
+                    and span._otel_span.status.status_code == StatusCode.UNSET
+                )
```

Comment:
I don't like that we're accessing the underlying `_otel_span` here -- would prefer to have `POTelSpan` manage the proxying as much as possible. We could add a `status` property to `POTelSpan` that returns `_otel_span.status` and use that here.

Reply:
Yes you are right. I thought this is done in multiple integrations and I will fix it in another Pr. But I guess I misremembered. 
Will fix it in this PR

Reply:
I have now implemented a status property. Have a look at the tests, because the Otel span statuses and the Sentry span status are different, the Sentry status one can get with the status property can be different from the one that has been set with set_status() before. And I default to "UNKNOWN_ERROR". Does this look ok to you?

Reply:
I think this makes sense.